rites/roman1969/__tests__/locales.test.ts

```diff
@@ -42,7 +42,7 @@ describe('Testing localization functionality', () => {
               expect(day?.name).toBe(d.value);
             } else {
               /* eslint-disable-next-line no-console */
-              console.log(
+              console.debug(
```

Comment:
This is an unrelated change.

Anyway, a proper fix would be to test all celebration names regardless whether they are celebrated in a year or not.

Reply:
i wasn't changing what was tested, and you're welcome to change the code i added if you like, these were things that made no sense to be plain logs. feel free to remove it.